### PR TITLE
Tweaks in benchmark script

### PR DESCRIPTION
**Pull request by lferran**
### Description
- Each worker gets assigned the same kbid to perform requests to
- Handle errors to inspect after test run
- Include nucliadb_performance in docker image so we can easily run scripts from within a cluster pod

### How was this PR tested?
Local tests

### CODE REVIEW

File: nucliadb_protos/rust/src/resources.rs [hunk 1]
```diff
@@ -414,8 +414,13 @@ pub struct QuestionAnswer {
     pub answer: ::prost::alloc::string::String,
     #[prost(string, tag="4")]
     pub answer_language: ::prost::alloc::string::String,
-    #[prost(int64, repeated, tag="5")]
```
Comment (tarekziade) [inline nucliadb_protos/rust/src/resources.rs:417]: unrelated change?

Comment (lferran) [reply nucliadb_protos/rust/src/resources.rs:417]: yeah, weird! I didn't think I touched this file. will remove the changes from the branch

Comment (tarekziade) [reply nucliadb_protos/rust/src/resources.rs:417]: if you rebase from `main` and run `make protos` hopefully it should be on par

File: nucliadb_performance/nucliadb_performance/test_search_cluster.py [hunk 2]
```diff
@@ -80,9 +109,37 @@ async def test_find(session):
...
+        detail = (
+            err.content and err.content.get("detail", None) if err.content else err.text
+        )
+        error = Error(
+            kbid=kbid,
+            endpoint=path.split("/")[-1],
+            status_code=err.status,
+            error=detail,
+        )
+        ERRORS.append(error)
```
Comment (tarekziade) [inline nucliadb_performance/nucliadb_performance/test_search_cluster.py:144]: That makes me think I could provide functions to print in the console (the right side is the errors) so you can also have them while running

**Review by tarekziade — APPROVED** [review]
